app/models/user.rb

```diff
@@ -592,7 +592,7 @@ def validate_mastodon_url
     return if mastodon_url.blank?
 
     uri = URI.parse(mastodon_url)
-    return if uri.host&.in?(Constants::ALLOWED_MASTODON_INSTANCES)
+    return if uri.host&.in?(Constants::AllowedMastodon::INSTANCES)
```

Comment:
what about `Constants::Mastodon::ALLOWED_INSTANCES` ?

Reply:
Hi @rhymes , in order to reduce the lines of  file `contents.rb`, I tried to seperate  it into two files: `constants/allowed_mastodon.rb` and `constants/site_config.rb`.

So `Constants::Mastodon::ALLOWED_INSTANCES` is actually the same as `Constants::ALLOWED_MASTODON_INSTANCES`.

Reply:
Hi @rhymes , `Constants::Mastodon::ALLOWED_INSTANCES` looks good, I will revise my PR. 
Thank you very much for your code review!

Reply:
Don't forget to mark the PR "ready for review" when it is :-)

Reply:
Thanks for reminding, I am waiting for CI finished. 😄

Reply:
there's a typo in the name of the constants, see https://travis-ci.com/github/forem/forem/jobs/370710544#L1603

Reply:
@rhymes Fixed the typo, please help to review again. 😃